repository-data/webfiles/src/main/resources/site/freemarker/common/document-list/services-list.ftl

```diff
@@ -0,0 +1,38 @@
+<#ftl output_format="HTML">
+<#include "../../include/imports.ftl">
+
+<@hst.setBundle basename="homepage.website.labels"/>
+<@fmt.message key="systems-and-services.title" var="title"/>
+<@fmt.message key="systems-and-services.text" var="text"/>
+<@fmt.message key="systems-and-services.subTitle" var="subTitle"/>
+<@fmt.message key="systems-and-services.buttonLabel" var="buttonLabel"/>
```

Comment:
I like that approach - I think we should start using the same approach and get all labels/headers at the top of the template. What do you think @ngta1, @jacsamell @ziemsky

Reply:
Hmmmm not sure, personally I like not having the extra variable but inserting directly into the element instead. If you have the variable `title` for example it's easy to get mixed up with `document.title`. It's also harder to tell from the template what the thing is referring to. If I see a variable in the FTL template I'm probably going to look in the component.java for the variable creation of `title`. I suppose we could name the variables `title-label` but personally I think you may as well insert directly.

I don't mind it being done the way it has, this is just my preference 😁